README.md

```diff
@@ -37,10 +37,11 @@ mesos_version: "1.0.1"
 
 # Debian
 mesos_package_version: "2.0.93"
-mesosphere_apt_url: "http://repos.mesosphere.com/{{ ansible_distribution | lower }}"
 mesos_os_distribution: "{{ ansible_distribution | lower }}"
 mesos_os_version: "{{ ansible_distribution_version.split('.') | join('') }}"
-mesos_apt_package: "mesos={{ mesos_version }}-{{ mesos_package_version }}.{{ mesos_os_distribution }}{{ mesos_os_version }}"
+mesos_apt_url: "http://{{ mesos_repo_host }}/{{ ansible_distribution | lower }}"
+mesos_package_full_version: "{{ mesos_version }}-{{ mesos_package_version }}.{{ mesos_os_distribution }}{{ mesos_os_version }}"
+mesos_apt_package: "mesos={{ mesos_package_full_version }}"
```

Comment:
let's add the `mesos_apt_pin_priority`(undefined by default)

Reply:
Undefined, because if you define it, then you have pinning deployed, which is something that the original playbook did not do.

If we do it, then we can have a much simpler deployment without the version in it, and it would be again a full fork.